unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerInfo/PassportPlayerInfoComponentController.cs

```diff
@@ -214,5 +233,30 @@ private void WhisperUser(string userId)
             socialAnalytics.SendStartedConversation(PlayerActionSource.Passport);
             OnClosePassport?.Invoke();
         }
+
+        private void UpdateFriendshipStatus(string userId, FriendshipAction action)
+        {
+            if (userId != currentPlayerId) return;
+            view.SetFriendStatus(ToFriendshipStatus(action));
+        }
+
+        private FriendshipStatus ToFriendshipStatus(FriendshipAction action)
+        {
+            switch (action)
+            {
+                case FriendshipAction.NONE:
+                case FriendshipAction.DELETED:
+                case FriendshipAction.REJECTED:
+                case FriendshipAction.CANCELLED:
+                default:
+                    return FriendshipStatus.NOT_FRIEND;
```

Comment:
maybe move at the end of the switch, so it will be not so obscure that CANCELLED is falling to default

Reply:
Done here: https://github.com/decentraland/unity-renderer/pull/3891/commits/063006a47c9df31e7dbbc8c6495bfb5edc346f9b